### PR TITLE
update zh_CN translation

### PR DESCRIPTION
**Pull request by JoveYu**

### CODE REVIEW

**Comment by zeagord** [comment]
cc @wu-sheng

**Comment by wu-sheng** [comment]
:) I will review this.

File: zipkin-ui/static/traces_zh_CN.properties [hunk 1]
```diff
@@ -1,28 +1,39 @@
 traces.servicename = 服务名
-traces.spanname = 跨度名
+traces.spanname = Span名
```
Comment (wu-sheng) [inline zipkin-ui/static/traces_zh_CN.properties]: Span名称

File: zipkin-ui/static/traces_zh_CN.properties [hunk 2]
```diff
@@ -1,28 +1,39 @@
...
+traces.lookback.3hours = 3 小时
+traces.lookback.6hours = 6 小时
+traces.lookback.12hours = 12 小时
+traces.lookback.1day = 1 天
+traces.lookback.2days = 2 天
+traces.lookback.7days = 7 天
+traces.lookback.custom = 自定义
 traces.stime = 开始时间
 traces.etime = 结束时间
+traces.annotationQuery = Annotation查询
```
Comment (wu-sheng) [inline zipkin-ui/static/traces_zh_CN.properties]: 根据Annotation查询

File: zipkin-ui/static/traces_zh_CN.properties [hunk 3]
```diff
@@ -1,28 +1,39 @@
...
 traces.duration = 持续时间 (μs) >=
 traces.limit = 数量
-traces.findBtn = 查找
 traces.sort = 排序
+traces.des = 请选择跟踪查找的条件。
+traces.findBtn = 查找
+traces.searchDisabled = 由于searchEnabled配置，禁止查询
 service-percentage-desc = 最大百分比优先
```
Comment (wu-sheng) [inline zipkin-ui/static/traces_zh_CN.properties]: 按百分比降序排列

File: zipkin-ui/static/traces_zh_CN.properties [hunk 4]
```diff
@@ -1,28 +1,39 @@
...
 service-percentage-asc = 最小百分比优先
```
Comment (wu-sheng) [inline zipkin-ui/static/traces_zh_CN.properties]: 按百分比升序排列

File: zipkin-ui/static/traces_zh_CN.properties (hunk 1 above)
Comment (JoveYu) [reply zipkin-ui/static/traces_zh_CN.properties]: 特殊名词  不希望带来误解，如果有专有名词的官方翻译，可以调整下

Comment (JoveYu) [reply zipkin-ui/static/traces_zh_CN.properties]: 特殊名词  不希望带来误解，如果有专有名词的官方翻译，可以调整下

File: zipkin-ui/static/traces_zh_CN.properties [hunk 5]
```diff
@@ -1,28 +1,39 @@
...
-traces.des = 请选择跟踪查找的条件。
 modal.title = 如何查找调用链
 modal.p1 = 如果你想查找调用链，那么你来对地方了！
 modal.title2 = 服务名
 modal.p2 = 首先，您可以选择要查找的服务。所有调用链都由服务名称进行索引。
 modal.p3 = 一旦选好了服务，你将有三个选项可供选择。见下面的注释：
-modal.title3 = 1. 通过跨度名查找
-modal.p4 = 跨度名通常是RPC方法名称或Rails端点。这允许您查找访问服务特定部分的调用链。
-modal.title4 = 2. 通过annotation查找
+modal.title3 = 1. 通过Span名查找
```
Comment (wu-sheng) [inline zipkin-ui/static/traces_zh_CN.properties]: 通过Span名称查找

File: zipkin-ui/static/traces_zh_CN.properties [hunk 6]
```diff
@@ -1,28 +1,39 @@
...
+modal.p4 = Span名通常是RPC方法名称或Rails端点。这允许您查找访问服务特定部分的调用链。
```
Comment (wu-sheng) [inline zipkin-ui/static/traces_zh_CN.properties]: Span名称通常是RPC方法名称或Rails端点。

@adriancole What is the original word? Rails?

**Review by wu-sheng — COMMENTED** [review]
My review inlines.

File: zipkin-ui/templates/trace.mustache [hunk 7]
```diff
@@ -4,7 +4,7 @@
     <div class='col-md-2'><div href='#'><strong data-i18n="trace.duration">Duration:</strong> <span class='badge badge-dark'>{{durationStr}}</span></div></div>
     <div class='col-md-2'><div href='#'><strong data-i18n="trace.service">Services:</strong> <span class='badge badge-dark'>{{serviceNameAndSpanCounts.length}}</span></div></div>
     <div class='col-md-2'><div href='#'><strong data-i18n="trace.depth">Depth:</strong> <span class='badge badge-dark'>{{depth}}</span></div></div>
-    <div class='col-md-2'><div href='#'><strong data-i18n="trace.spans.length">Total Spans:</strong> <span class='badge badge-dark'>{{spans.length}}</span></div></div>
+    <div class='col-md-2'><div href='#'><strong data-i18n="trace.spanCount">Total Spans:</strong> <span class='badge badge-dark'>{{spans.length}}</span></div></div>
```
Comment (codefromthecrypt) [inline zipkin-ui/templates/trace.mustache:7]: oops thanks!

File: zipkin-ui/static/traces_zh_CN.properties (hunk 1 above)
Comment (wu-sheng) [reply zipkin-ui/static/traces_zh_CN.properties]: Span 名称。Span是不需要翻译的。

File: zipkin-ui/static/traces_zh_CN.properties (hunk 6 above)
Comment (JoveYu) [reply zipkin-ui/static/traces_zh_CN.properties]: original world is: Span names are generally RPC method names or Rails endpoints. This allows you to look up traces that accessed a particular part of the service.

Comment (JoveYu) [reply zipkin-ui/static/traces_zh_CN.properties]: maybe is Ruby on Rails endpoints

**Comment by JoveYu** [comment]
Fix above

**Review by wu-sheng — APPROVED** [review]

**Comment by codefromthecrypt** [comment]
Thank you very much @JoveYu and @wu-sheng 

I assume right now the page is broken for chinese, so I will cut a release

**Pull request merged**